datastore/datastore_quickstart/main.go

```diff
@@ -0,0 +1,53 @@
+// Copyright 2016 Google Inc. All rights reserved.
+// Use of this source code is governed by the Apache 2.0
+// license that can be found in the LICENSE file.
+
+// [START datastore_quickstart]
+// Sample datastore_quickstart fetches an entity from Google Cloud Datastore.
+package main
+
+import (
+	"fmt"
+	"golang.org/x/net/context"
```

Comment:
Move the context import to the next group. The uppermost group belongs to the standard library imports.